uportal-war/src/main/java/net/sf/ehcache/distribution/jgroups/BootstrapRequest.java

```diff
@@ -0,0 +1,161 @@
+/**
+ *  Copyright 2003-2010 Terracotta, Inc.
```

Comment:
How @ChristianMurphy?  I'm certain we can add it as a dependency if it's creators have made it available -- in binary format -- in Maven Central.  If it only exists as source, perhaps we can learn from it and write our own?